### PR TITLE
use deepequal on nbt for Item.equal

### PR DESCRIPTION
**Pull request by u9g**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,6 +35,9 @@
   },
   "homepage": "https://github.com/PrismarineJS/prismarine-item#readme",
   "dependencies": {
+    "fast-deep-equal": "^3.1.3",
+    "install": "^0.13.0",
```
Comment (rom1504) [inline package.json:39]: remove

Comment (u9g) [reply package.json]: done

File: test/basic.test.js [hunk 2]
```diff
@@ -228,3 +228,26 @@ describe('test anvil functions', () => {
...
+    expect(Item.equal(itemOne, itemTwo)).toStrictEqual(false)
+  })
+  test('two unenchanted', () => {
+    const itemOne = new Item(mcData.itemsByName.diamond_sword.id, 1)
+    const itemTwo = new Item(mcData.itemsByName.diamond_sword.id, 1)
+    expect(Item.equal(itemOne, itemTwo)).toStrictEqual(true)
+  })
+  test('two enchanted', () => {
+    const itemOne = new Item(mcData.itemsByName.diamond_sword.id, 1)
+    itemOne.enchants = [{ name: 'sharpness', lvl: 5 }]
```
Comment (rom1504) [inline test/basic.test.js:248]: can you add one test with 2 properties in opposite orders (should equal true) ?

Comment (u9g) [reply test/basic.test.js:248]: done

**Review by rom1504 — APPROVED** [review]

**Pull request merged**